lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2287,24 +2287,26 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
             }
             self.file._annotations[-1][1].append(link_annotation)
 
-        # If fonttype != 3 or there are no multibyte characters, emit the whole
-        # string at once.
-        if fonttype != 3 or all(ord(char) <= 255 for char in s):
+        # If fonttype != 3 emit the whole string at once without manual
+        # kerning.
+        if fonttype != 3:
             self.file.output(Op.begin_text,
                              self.file.fontName(prop), fontsize, Op.selectfont)
             self._setup_textpos(x, y, angle)
-            self.file.output(self.encode_string(s, fonttype), Op.show,
-                             Op.end_text)
+            self.file.output(self.encode_string(s, fonttype),
+                             Op.show, Op.end_text)
 
         # There is no way to access multibyte characters of Type 3 fonts, as
         # they cannot have a CIDMap.  Therefore, in this case we break the
         # string into chunks, where each chunk contains either a string of
-        # consecutive 1-byte characters or a single multibyte character.  Each
-        # chunk is emitted with a separate command: 1-byte characters use the
-        # regular text show command (Tj), whereas multibyte characters use
-        # the XObject command (Do).  (If using Type 42 fonts, all of this
-        # complication is avoided, but of course, those fonts can not be
-        # subsetted.)
+        # consecutive 1-byte characters or a single multibyte character.
+        # A sequence of 1-byte characters is broken into multiple chunks to
+        # adjust the kerning between adjacent chunks.  Each chunk is emitted
+        # with a separate command: 1-byte characters use the regular text show
+        # command (TJ) with appropriate kerning between chunks, whereas
+        # multibyte characters use the XObject command (Do).  (If using Type
+        # 42 fonts, all of this complication is avoided, but of course, those
+        # fonts can not be subsetted.)
```

Comment:
This is a bit misleading as mentioned by @anntzer in https://github.com/matplotlib/matplotlib/issues/11303#issuecomment-391817482

Until we decide on https://github.com/matplotlib/matplotlib/pull/18143, maybe this would be a better way to put it?
```suggestion
        # multibyte characters use the XObject command (Do).  (If using Type
        # 42 fonts, all of this complication is avoided, but of course, subsetting
        # those fonts is complex/hard to be implemented.)
```

Reply:
But I double with @anntzer that it's not a necessary change:
1. For somebody without the knowledge of fontTools, the original statement makes sense.
2. For somebody with the knowledge of subsetting, the original statement won't be a blocker.

Reply:
I understood that the @anntzer's statement was only referring to kerning, however, I've made the statement about subsetting more precise as you've suggested.